lib/card-types.js

```diff
@@ -134,7 +134,8 @@ var cardTypes = {
     type: 'maestro',
     patterns: [
       493698,
-      [500000, 506698],
+      [5000, 5018],
+      [502000, 506698],
```

Comment:
I split the patterns here to make `5019` Dankort-exclusive. This didn't affect the other unit tests for Maestro.